City-of-Helsinki/employment-services-ui
# Thf 631 lang attribut

changed lang attribute handling

#### bug not working

- Copy the URL and paste it to new tab.
- Open the dev tools again and see that your lang attribute is same as your selected filter. 
- Select the local language 
- You should get the local language to the lang attribute (not happening :( )

Test with back end https://github.com/City-of-Helsinki/drupal-employment-services/pull/177


### testing

- page http://localhost:3000/ajankohtaista/tapahtumat
- Choose some language from language select
- Open browser dev tools. Your lang attribute should be same as your selected language.
- Copy the URL and paste it to new tab.
- Open the dev tools again and see that your lang attribute is same as your selected filter.
